hibernate-core/src/main/java/org/hibernate/boot/internal/SessionFactoryBuilderImpl.java

```diff
@@ -519,6 +520,7 @@ public SessionFactory build() {
 		private boolean wrapResultSetsEnabled;
 
 		private Map<String, SQLFunction> sqlFunctions;
+		private boolean preferUserTransaction;
```

Comment:
Why isn't this exposed then on the `SessionFactoryBuilder` contract to be set?  Reading the config values from the configuration settings in the ctor is just initialization.